### PR TITLE
#167697967  Add footer layout

### PR DESCRIPTION
**Pull request by tohbay**
#### What does this PR do?
Add footer layout for the application

#### Description of Task to be completed?
- ensure there is a mobile responsive footer layout
- ensure there is a desktop footer layout

#### How should this be manually tested?
1. Clone the repo
2. On your terminal, run `npm install`
3. On your terminal run `npm run start:dev`

#### Any background context you want to provide?
Ensure that there is node, npm and google chrome browser installed on your machine.

#### What are the relevant pivotal tracker stories? (if applicable)
#167697967: https://www.pivotaltracker.com/story/show/167697967

### CODE REVIEW

**Review by Onyimatics — APPROVED** [review]
Nice work @tobechukwuobitube. Looks good to me

File: tests/Footer.spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { shallow } from './enzyme';
+import Footer from '../src/components/Footer/Footer';
+
+describe('Footer test', () => {
+  it('should work fine on Footer', () => {
```
Comment (segunolalive) [inline tests/Footer.spec.js]: Change this to: `it should render without crashing`.

File: src/components/Footer/Footer.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:400,500,600,700,800,900|Roboto&display=swap');
+
```
Comment (segunolalive) [inline src/components/Footer/Footer.css:2]: Use custom properties (CSS variables) for properties that change often. e.g:

```css
:root {
  --font: 'Poppins', sans-serif;
  font-family: var(--font);
  font-style: normal;
}

.someSelectorWithRobotoText {
  --font: Roboto;
}
```

There's a base font-size already set by tailwind, use `rem` or `em` values in your code base for font-sizes and pixel widths. e.g

```css
.selector {
  padding: 4px;
  /* becomes */
   padding: 0.25rem; 
   /* assuming 1rem = 16px */
}
```

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './Footer.css';
+import FacebookIcon from '../../assets/@icons/facebook';
+import TwitterIcon from '../../assets/@icons/twitter';
+import LinkedInIcon from '../../assets/@icons/linkedin';
+import InstagramIcon from '../../assets/@icons/instagram';
+import YoutubeIcon from '../../assets/@icons/youtube';
```
Comment (segunolalive) [inline src/components/Footer/Footer.js]: Import the SVG directly. No need to create new components.

```js
import { ReactComponent as YoutubeIcon } from '../../assets/@icons/youtube';

```

**Review by segunolalive — COMMENTED** [review]
Looks good. make the required changes.

File: tests/Footer.spec.js (hunk 1 above)
Comment (tohbay) [reply tests/Footer.spec.js]: Thank you for this feedback. I will revert immediately.

**Pull request merged**